reagurk/public/index.html

```diff
@@ -10,7 +10,6 @@
     <meta property="twitter:title" content="Gurkult" />
     <meta property="og:site_name" content="Gurkult" />
     <meta property="og:locale" content="en_US" />
-    <meta property="og:description" content="The Gurkult is an open source community website."/>
     <meta name="description" content="The Gurkult is an open source community website." />
```

Comment:
This should be `og:description` right?

Reply:
@Arnav-2004 earlier it was og: description but the other guy asked me to changed this should i do og: description again?

Reply:
@RishiKumarRay I am pretty sure they said to remove the duplicates.

Reply:
actually I think it's fine.

Reply:
@Arnav-2004 Yeah , True , maybe i assumed something else , sorry for that also i have updated this now its og: description please review and merge , Thanks

Reply:
@Arnav-2004 I updated it , let me know if you want to remove it .

Reply:
`description` and `og:description` are both necessary.

Reply:
@dawnofmidnight I updated to og: description please review if there are more changes to be